include/log.h

```diff
@@ -25,40 +19,4 @@ namespace celerity::detail {
 
 using log_level = spdlog::level::level_enum;
 
-struct log_context {
```

Comment:
What happened to the log context? Was it unused?

Reply:
It's only use was in `worker_job`, which is now gone. `executor` has full control of submissions an does all the relevant logging.